scripts/gbmake

```diff
@@ -72,9 +72,9 @@ gnubee_defconfig() {
 	case $mach in
 	    "" | defconfig ) # guess
 		if mach=`gnubee_model`; then
-		    echo Using config $mach
+		    echo "I: Using config $mach"
```

Comment:
I'm no fan of those one-letter prefixes, what does the I stand for ? E or W i can guess...
But I also prefer the longer, more explicit ones.

Reply:
I: is info.  Doing this is consistent with the messages that the debian installer produces so it certainly makes sense  for the messages generated by the "config" script.
It is less obvious that it would be appropriate for the gbmake script, but maybe it does and I'm certainly open to making it clearer.

Reply:
OK debian consistency is an accepted excuse ;-)